org.lflang.ui/src/org/lflang/ui/highlighting/LFSemanticHighlightingCalculator.java

```diff
@@ -61,23 +56,6 @@ public void provideHighlightingFor(XtextResource resource, IHighlightedPositionA
             return;
         }
         Model model = (Model) parseResult.getRootASTElement();
-        
-        // Provide keyword highlighting for special mode transitions
```

Comment:
How come this is no longer necessary?

Reply:
When there was only `reset(<mode>)` and `continue(<mode>)`, I was able to make a workaround that treated these keywords as strings in the grammar (#947) and then faked their role as keywords in the editor by adjusting the highlighting (which also only worked in Epoch). However, with `reset` now used in the enum rule for built-in triggers and as boolean modifier (reset?='reset') in state variables, it was not possible to treat is as a non-keyword. So I also rolled back the workaround in transition effect to make the grammar more consistent.